packages/drivers/odsp-driver/src/test/zipItDataRepresentationTests.spec.ts

```diff
@@ -139,4 +144,46 @@ describe("Tree Representation tests", () => {
         node2.addString("fourth");
         validate(2 + 5 + 2 + 2 + 6 + 2 + 5 + 2 + 2 + 6);
     });
+
+    it("blob instance test", async () => {
+        const blobNode = new BlobShallowCopy(new ReadBuffer(new Uint8Array()), 0, 0);
+        assertBlobCoreInstance(blobNode);
```

Comment:
should string be required for assertBlobCoreInstance()? In general, I do not like it for asserts (as we should strive for code be readable / self-explanatory and require fewer comments), but while we do not have tags, it feels like string is required, no?